### DISCUSSION TITLE
Improve API of component Message (docs)

### DISCUSSION THREAD

**Discussion by evgeniyworkbel** [ant-design/ant-design#42051] [Ideas]
Hello. i noticed that in [docs](https://ant.design/components/message#api) argument "type" is not included in API of component though it is performed in the second example ("Other types of message") - there this argument as obj propety is passed in config message.open().

Please improve api docs of this component. Many thanks!

**Comment by zombieJ** [comment]
do you mean follow after that?
 
<img width="451" alt="截屏2023-04-27 22 52 43" src="https://user-images.githubusercontent.com/5378891/234901246-ac3efd83-1431-47ef-99ff-773f45929c31.png">

**Reply by evgeniyworkbel** [reply]
> do you mean follow after that?
> 
> <img alt="截屏2023-04-27 22 52 43" width="451" src="https://user-images.githubusercontent.com/5378891/234901246-ac3efd83-1431-47ef-99ff-773f45929c31.png">

Yes, I do

**Reply by lionxiong** [reply]
So do I,where can I put message.config() ?
